lesson_2_Section_30_new.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
<p>Yes this outline is exactly right.  Planning on following this exactly.  The last point </p><p> </p><blockquote>What a full GP would look like is this but as you can see there's a lot more going on. Thatll be for the next lesson</blockquote><p> </p><p>is a great place to end</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bwengals/etc/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>